src/openrct2/PlatformEnvironment.h

```diff
@@ -16,18 +16,18 @@
 
 namespace OpenRCT2
 {
-    enum class DIRBASE : size_t
+    enum class Dirbase : size_t
```

Comment:
I wonder if `DirBase` would be better than `Dirbase`. It seems like it's two words, to me.

Some tests will still need to be adjusted to account for this change.

Reply:
Was wondering this myself. I’ll go with DirBase, then.